### PR TITLE
change tf version and change BUILD rule

### PR DESCRIPTION
**Pull request by ziyi-yang**

### CODE REVIEW

**Comment by ziyi-yang** [comment]
@xiangxin-zhu @caihsiao

File: tensorboard/BUILD [hunk 1]
```diff
@@ -25,7 +25,8 @@ py_binary(
     deps = [
         ":default",
         ":program",
-        "//tensorboard:expect_tensorflow_installed",
+        #"//tensorboard:expect_tensorflow_installed",
```
Comment (xiangxin-zhu) [inline tensorboard/BUILD]: remove #"//tensorboard:expect_tensorflow_installed",

File: WORKSPACE [hunk 2]
```diff
@@ -46,19 +46,37 @@ load("@io_bazel_rules_closure//closure:defs.bzl", "closure_repositories")
...
+load("//build_rules:repo.bzl", "tensorflow_http_archive")
+
+tensorflow_http_archive(
+  name = "org_tensorflow",
+  sha256 = "cbd96914936ce3aacc39e02c2efb711f937f8ebcda888c349eab075185d7c914",
+  git_commit = "d8fac4cb80eb0c42d2550bcb720a80d29fc5f22d",
 )
 
-load("@org_tensorflow//tensorflow:workspace.bzl", "tf_workspace")
+#http_archive(
```
Comment (xiangxin-zhu) [inline WORKSPACE]: remove the lines that are commented out.

**Review by xiangxin-zhu — APPROVED** [review]

File: WORKSPACE (hunk 2 above)
Comment (ziyi-yang) [reply WORKSPACE]: Removed.

File: tensorboard/BUILD (hunk 1 above)
Comment (ziyi-yang) [reply tensorboard/BUILD]: Removed.

**Review by ziyi-yang — COMMENTED** [review]
Removed.

**Pull request merged**